palisade-service/src/main/java/uk/gov/gchq/palisade/service/palisade/request/AddResourceRequest.java

```diff
@@ -25,33 +28,35 @@
 
 import static java.util.Objects.requireNonNull;
 
-public class ReadRequest extends Request {
-    private String token;
+/**
+ * This class is used to request that details about a resource is added to the resource-service.
+ */
+@JsonIgnoreProperties(value = {"originalRequestId"})
+public class AddResourceRequest extends Request {
```

Comment:
Why is the ReadRequest being changed to an AddResourceRequest? Is this a git thing as well?

Reply:
Git thing